docs/api/geo_datasets.csv

```diff
@@ -1,17 +1,18 @@
 Dataset,Type,Source,Size (px),Resolution (m)
-`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","~40,000x40,000",~30
+`Aboveground Woody Biomass`_,Mask,"Landsat, LiDAR","40,000x40,000",30
 `Aster Global DEM`_,Mask,Aster,"3,601x3,601",30
-`Canadian Building Footprints`_,Labels,Generated,,
```

Comment:
"Labels" is ambiguous. When I hear "labels" I think classification or regression. I think "geometries" is more clear for this dataset. Also, "generated" doesn't tell me anything.

Reply:
"generated" to me is model output vs. manual interpretation